README.md

```diff
@@ -15,6 +15,8 @@ The following folders can be found in this repository:
 - **[`metriccentury`](https://github.com/deephaven/examples/tree/main/metriccentury)** - Data recorded from a 100 km bike ride
 - **[`pems`](https://pems.dot.ca.gov/)** - Traffic flow data collected near Davis, CA.
 - **[`taxi`](https://azure.microsoft.com/en-us/services/open-datasets/catalog/nyc-taxi-limousine-commission-yellow-taxi-trip-records/)** - Yellow Taxi trip records
+- **[`fit`](https://www.strava.com/) - Workout results in the proprietary fit format developed by Garmin. Downloadable from e.g. Strava
```

Comment:
```suggestion
- **[`fit`](https://www.strava.com/) - Workout results in the proprietary fit format developed by Garmin. Downloadable from Strava.
```

Reply:
Can we provide a direct link to the data to download? Or is the point YOU the user have the data?

Reply:
Yes, the user has their own data under their own logins - that is kindof the point.